modules/ghlint-cli/src/main/kotlin/net/twisterrob/ghlint/cli/CLI.kt

```diff
@@ -26,14 +27,39 @@
 	name = "ghlint",
 ), Configuration {
 
+	// IMPORTANT: The order of properties is important, it'll define how "--help" is printed.
+
+	override val files: List<Path> by argument()
+		.path(mustExist = true, canBeDir = false)
+		.multiple(required = false)
+		.help("Workflow YML files to check for problems.")
+
+	private val inputs: InputOptions by InputOptions()
+
+	override val root: Path by option("--root")
+		.path(mustExist = true, canBeDir = true, canBeFile = false)
+		.default(Path.of("."))
+		.help("Root directory of the repository.")
+
+	override val isVerbose: Boolean by option("--verbose")
+		.flag(default = false, defaultForHelp = "off")
+		.help("Prints more information.")
+
 	@Suppress("detekt.ClassOrdering")
 	override fun run() {
+		if (files.isEmpty()) {
+			System.err.println("Error: Missing argument <files>")
```

Comment:
This error handling approach mixes business logic with CLI concerns. Consider moving this validation to a separate validation method or using Clikt's built-in validation mechanisms.